### PR TITLE
Feature recommendation

### PR DESCRIPTION
**Pull request by Surerus**
Many endpoints, services and entities modification (technology)

### CODE REVIEW

File: src/main/java/com/ciandt/techgallery/persistence/model/Technology.java [hunk 1]
```diff
@@ -28,17 +31,20 @@
...
   @Unindex
   private String author;
-
+
   @Unindex
   private String image;
-
+
+  @Unindex
+  private String citRecommendation;
```
Comment (larruda) [inline src/main/java/com/ciandt/techgallery/persistence/model/Technology.java]: Let's try not to use CI&T names and/or references whenever possible.

File: src/main/java/com/ciandt/techgallery/service/model/TechnologyResponse.java [hunk 2]
```diff
@@ -1,72 +1,92 @@
...
+  public void setImage(String image) {
+    this.image = image;
+  }
+
+  public String getCitRecommendation() {
+    return citRecommendation;
+  }
+
+  public void setCitRecommendation(String citRecommendation) {
+    this.citRecommendation = citRecommendation;
```
Comment (larruda) [inline src/main/java/com/ciandt/techgallery/service/model/TechnologyResponse.java]: Same as above.

File: src/main/java/com/ciandt/techgallery/persistence/model/Technology.java (hunk 1 above)
Comment (Surerus) [inline src/main/java/com/ciandt/techgallery/persistence/model/Technology.java]: Done.

File: src/main/java/com/ciandt/techgallery/service/model/TechnologyResponse.java (hunk 2 above)
Comment (Surerus) [inline src/main/java/com/ciandt/techgallery/service/model/TechnologyResponse.java]: Done.

**Pull request merged**